evap/evaluation/migrations/0127_fix_logentry_types.py

```diff
@@ -0,0 +1,23 @@
+import importlib
+from django.db import migrations
+
+original_migration_module = importlib.import_module("evap.evaluation.migrations.0123_evaluation_state_fsm_int")
+
+
+def noop(_apps, _schema_editor):
+    pass
```

Comment:
Django already has this: https://docs.djangoproject.com/en/4.0/ref/migration-operations/#django.db.migrations.operations.RunPython.noop